commands/version/README.md

```diff
@@ -263,6 +264,26 @@ When run with this flag, `lerna version` will force publish the specified packag
 
 > This will skip the `lerna changed` check for changed packages and forces a package that didn't have a `git diff` change to be updated.
 
+### `--git-tag-command <cmd>`
+
+Allows users to specify a wrapper command in CD/CI pipelines that have no direct write access.
```

Comment:
```suggestion
Allows users to specify a custom command to be used when applying git tags. For example, this may be useful for providing a wrapper command in CI/CD pipelines that have no direct write access.
```